doc/source/whatsnew/v1.4.0.rst

```diff
@@ -639,6 +639,7 @@ I/O
 - Bug in :func:`read_csv` with :code:`float_precision="round_trip"` which did not skip initial/trailing whitespace (:issue:`43713`)
 - Bug in dumping/loading a :class:`DataFrame` with ``yaml.dump(frame)`` (:issue:`42748`)
 - Bug in :func:`read_csv` raising ``ValueError`` when ``parse_dates`` was used with ``MultiIndex`` columns (:issue:`8991`)
+- Bug in :func:`read_csv` raising ``AttributeError`` when attempting to read .csv files and infer index column dtype (:issue:`44079`)
```

Comment:
this needs to specifically mention using an nullable integer type (as this is not generic rather a specific thing)

Reply:
done